### PR TITLE
Support for Splunk 7.0.0

### PR DESCRIPTION
**Pull request by mchene**
Also added updates for
1. Querying Node and Service meta data in docker swarm mode
2. Updated latest docker client to 1.17
3. Fix bug with docker tops formatting
4. Fix bug with lookup CSV, now using .csv postfix

### CODE REVIEW

**Comment by matthewmodestino** [comment]
Cleaned up the compose typos and looks good so far in my lab. Attached the v3.3 compose file I used. 

[docker-compose-withuf-v3.txt](https://github.com/splunk/docker-itmonitoring/files/1412945/docker-compose-withuf-v3.txt)

**Comment by domeger** [comment]
Still pulling image splunk/splunk:6.5.3-monitor. Did you want to push and rebuild the image?

**Comment by matthewmodestino** [comment]
I used the build-enterprise.sh to build 7.0.0-monitor locally, then deployed. I assume we were just waiting for testing before push?

**Comment by domeger** [comment]
@matthewmodestino  - We will need to push the image to DockerHub and then change the image on the Compose file then we can test it.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-  vsplunk:
-    image: busybox
-    volumes:
-      - /opt/splunk/etc
-      - /opt/splunk/var
   splunkenterprise:
     #build: .
     hostname: splunkenterprise
-    image: splunk/splunk:6.5.3-monitor
+    image: splunk/splunk:7.7.0-monitor
```
Comment (tauliang) [inline docker-compose.yml]: Should this be 7.0.0-monitor?

Comment (mchene) [reply docker-compose.yml]: yes it should.  good catch!  Changing it now.

**Comment by mchene** [comment]
Support for Splunk Enterprise and UF 7.0.0, bug fixes and added feature updates to now collect node and service meta-data from docker swarm mode deployments.

**Pull request merged**